aws/resource_aws_ssm_parameter.go

```diff
@@ -47,35 +51,47 @@ func resourceAwsSsmParameter() *schema.Resource {
 			"key_id": {
 				Type:     schema.TypeString,
 				Optional: true,
-				ForceNew: true,
```

Comment:
Is `key_id` modifiable now? Can you please update `TestAccAWSSSMParameter_secure_with_key` to test this?

Reply:
Yes it works well, I tested it manually but I'll be adding tests of course.